contributing_to_docs/doc_guidelines.adoc

```diff
@@ -61,7 +61,8 @@ would be appropriate for a topic titled Managing Authorization Policies.
 a|- Use title case in all topic titles and section headings. See
 http://titlecase.com/ for more information.
 - Try to be as descriptive as possible with the topic title or section headings
-without making them unnecessarily too long.
+without making them unnecessarily too long. Use a verb form in headings (for example,
+Creating, Managing, Using, etc.) whenever possible.
```

Comment:
There's been some casual talk recently about not having "Overview" sections (as we've specifically known/required them in our guidelines previously) just to tick a box. Though of course there are definite places for overview-style material. Agree that we should try to define a bit what that sort of overview material should look like.

Deferring to @aheslin though re: the "Overview" topic requirement specifically; I think we should try to get those sentiments down on paper or at least hash it out a little more (seems like a good time w/ these guidelines updates).